src/main/java/seedu/address/model/person/Github.java

```diff
@@ -0,0 +1,59 @@
+package seedu.address.model.person;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.util.AppUtil.checkArgument;
+
+/**
+ * Represents a Person's GitHub profile in the address book.
+ * Guarantees: immutable; is valid as declared in {@link #isValidGh(String)}
+ */
+public class Github {
+
+    public static final String MESSAGE_CONSTRAINTS = "Github profiles can take any values "
+            + "(whether name or link to Github profile page, and it should not be blank";
```

Comment:
I believe that there should be a close bracket in this line
```suggestion
            + "(whether name or link to Github profile page, and it should not be blank)";
```